pcs_and_grm.R

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(SeqArray)
+library(GENESIS)
+library(SeqVarTools)
+library(magrittr)
+library(SNPRelate)
+
+sessionInfo()
+
+# read arguments
+argp <- arg_parser("Generate PCs and GRM") %>%
+  add_argument("gds_file", help = "GDS file") %>%
+  add_argument("--out_prefix", help = "Prefix for output files") %>%
+  add_argument("--variant_id", help = "File with vector of variant IDs") %>%
+  add_argument("--sample_id", help = "File with vector of sample IDs") %>%
+  add_argument("--kin.thresh", default = 5.5,
+               help = "Kinship threshold for pcair (2 ^ -kin.thresh)") %>%
+  add_argument("--n_pcs", default = 3,
```

Comment:
We usually don't know what value to use for `n_pcs` until after we've run `pcair` and looked at plots to see which PCs separate known populations. The TOPMed pipeline has separate scripts for pcair and pcrelate for this reason. It does seem nice and compact to combine it all in the same script, but I don't know if there's another way to address that issue.